homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
Also, the MRO says ZWaveDeviceEntity will always be locked up before light, as MRO is deep first, left to right, good head.